generator/internal/language/rust.go

```diff
@@ -1060,6 +1060,10 @@ func escapeUrls(line string) string {
 			// Skip adding <> if the url is already surrounded by angled brackets.
 			escapedLine.WriteString(line[lastIndex:match[1]])
 			lastIndex = match[1]
+		} else if strings.Contains(line[lastIndex:match[0]], "<a href") {
```

Comment:
This is good, but it is growing in complexity.  Next time we should consider #763 .

Reply:
Thanks! I didn't realize goldmark had an extension for this. Might be more robust than manually processing every type of url. Will look into this in a follow up PR.